src/room/RTCEngine.ts

```diff
@@ -15,6 +15,7 @@ import { sleep } from './utils';
 const lossyDataChannel = '_lossy';
 const reliableDataChannel = '_reliable';
 const maxReconnectRetries = 5;
+const maxReconnectingTimeout = 2 * 1000;
```

Comment:
do you think a minimum wait for 1s is sufficient to allow connected to flip to connecting?

```suggestion
const minReconnectWait = 1 * 1000;
```

Reply:
I'm not sure, 2s just a initial value. need more observe